src/Protocol/Protocol_1_8.cpp

```diff
@@ -1587,6 +1587,26 @@ void cProtocol_1_8_0::SendUseBed(const cEntity & a_Entity, int a_BlockX, int a_B
 
 
 
+void cProtocol_1_8_0::SendUnlockRecipe(UInt32 a_RecipeID)
+{
+	// Not implemented
```

Comment:
Is it "not implemented in Cuberite" or rather "Client doesn't support this feature"?

Reply:
'Client doesn't support this feature', but I couldn't find a way to only define it in `Protocol_1_12`